### DISCUSSION TITLE
Creating a shopping cart

### DISCUSSION THREAD

**Discussion by cos011** [Budibase/budibase#11670] [Help]
Good afternoon. My task is to create an online store application. There were difficulties when creating the process of buying a product, sending it to the cart.

There are two tables in the postgresql database. The first table contains the name of the goods, the second table contains their specifications, both tables are linked by the "One to Many" relationship.

I have added elements on the screen:
A list of products, then when the user clicks on a specific product, it becomes possible to select its specification from the second table.

Problem

It is necessary to create a shopping cart and ensure that after the user selects the product and its specification, this product with its specification is added to the cart, then after the user clicks the "buy" button, the contents of the basket with goods and their specification must be sent by email.

I ask for help in creating an online store shopping cart! Help please.

**Comment by melohagan** [comment]
Hey @cos011 

You can send an email using automations, for example: https://docs.budibase.com/docs/conditions-1

**Comment by cos011** [comment]
Good day.
The problem is not that it is not possible to send the result to the mail. I can't implement this specifically:

It is necessary to create a shopping cart page and make sure that after the user selects the product and its specification, this product with its specification will be added to this cart, then, after the user clicks the "buy" button, the contents of the basket with the products and their specification should be sent by email.

How to implement it? As I understand it, when a user selects a product, I need to write the name and specifications of his choice into a variable? Next, write the value of this variable to a temporary table and send it to the mail via atomization?

**Reply by melohagan** [reply]
Hey @cos011 I'm currently in the process of writing up some docs for a similar example, which I'll link when finished.

**Reply by cos011** [reply]
Hi. I'm really looking forward to this instruction

**Reply by cos011** [reply]
Hi. No information about when the instruction will be available?

**Comment by melohagan** [comment]
Hey @cos011 Hopefully this gives you some inspiration! https://docs.budibase.com/docs/send-email
